pages/Posts.tsx

```diff
@@ -125,24 +124,31 @@ export default function Posts() {
               <HiDotsHorizontal />
             </div>
             <h4 className="text-1xl">{post.body}</h4>
-            <div className="flex justify-between items-center pt-2">
-              <button
-                type="button"
-                className="flex items-center justify-center"
-                onClick={(event) => handleLikeClick(event, post)}
-              >
-                {post.liked ? <FaHeart size="20" /> : <FaRegHeart size="20" />}
-                {post.likes === 0 ? (
-                  <span className="hidden">{post.likes}</span>
-                ) : (
-                  <span className="pl-4">{post.likes}</span>
-                )}
-              </button>
-
+            <div className="flex justify-between pt-2">
+              <div>
+                <button
+                  type="button"
+                  className="flex items-center justify-center"
+                  onClick={(event) => handleLikeClick(event, post)}
+                >
+                  {post.liked ? (
+                    <FaHeart size="20" />
+                  ) : (
+                    <FaRegHeart size="20" />
+                  )}
+                  {post.likes === 0 ? (
+                    <span className="invisible ml-2">{post.likes}</span>
+                  ) : (
+                    <span className="ml-2">{post.likes}</span>
+                  )}
+                </button>
+              </div>
+              <div>
+                <button type="button">
+                  <FaRegComment />
+                </button>
```

Comment:
There is no change related to the share button here, I am not sure why it is part of the commit and PR message. Having the share in the PR description is confusing